Changelog.md

```diff
@@ -6,7 +6,7 @@ Language Features:
 Compiler Features:
  * Build System: LLL is not built anymore by default. Must configure it with CMake as `-DLLL=ON`.
  * Code generator: Do not perform redundant double cleanup on unsigned integers when loading from calldata.
-
+ * Optimiser: Remove redundant `virtual` when function is declared with `override`.
```

Comment:
I don't think we need a Changelog entry for this PR.

Reply:
👍